iron-selectable.html

```diff
@@ -336,10 +346,19 @@
       if (this.selectedAttribute) {
         this.toggleAttribute(this.selectedAttribute, isSelected, item);
       }
+      if (this.selectedPropertyValue) {
+        item[this.selectedAttribute] = this.selectedPropertyValue;
+      }
```

Comment:
Seems like the assignment of the value should be contingent on if selectedAttribute exists, and also probably needs to check if isSelected and if !isSelected then set to null (or something?).. eg:

```
if (this.selectedAttribute) {
    if (this.selectedPropertyValue) {
        item[this.selectedAttribute] = isSelected ? this.selectedPropertyValue : null;
    }
    else {
        this.toggleAttribute(this.selectedAttribute, isSelected, item);
    }
}
```